packages/mdc-select/_mixins.scss

```diff
@@ -108,7 +108,12 @@
 }
 
 @mixin mdc-select-outline-shape-radius($radius, $rtl-reflexive: false) {
-  $resolved-radius: mdc-shape-resolve-percentage-radius($mdc-select-height, $radius);
+  $resolved-radius: nth(mdc-shape-resolve-percentage-radius($mdc-select-height, mdc-shape-prop-value($radius)), 1);
+
+  @if (length(mdc-shape-prop-value($radius)) > 1) {
+    // stylelint-disable-next-line max-line-length
+    @warn "mdc-select-outline-shape-radius currently only supports a single radius value. Follow https://github.com/material-components/material-components-web/issues/4140 for updates.";
```

Comment:
Is there a way to put this on a new line? Its difficult to see this on my terminal:
<img width="1640" alt="Screen Shot 2019-03-28 at 12 09 13 PM" src="https://user-images.githubusercontent.com/579873/55185807-5cf15a00-5152-11e9-8d6e-f7e50e3083cb.png">

Reply:
I've shortened the message as much as possible. I originally tried 2 separate `@warn` statements but that ends up printing the stack trace each time, which makes it effectively more confusing to read. I also tried `\n` which doesn't work as a newline.